### PR TITLE
Add php quickstart

### PR DESCRIPTION
**Pull request by Tonnu**

### CODE REVIEW

File: content/quickstarts/building/php.md [hunk 1]
```diff
@@ -1,2 +1,339 @@
...
+
+```
+$ php -S 127.0.0.1:8000
+```
+
+Now in your browser navigate to `127.0.0.1:8000` and you should be
+presented with the following json:
+
+```
+"{'cities':'San Francisco, Amsterdam, Berlin, New York, Tokyo, London'}"
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: This json does not matches the output from the sample project: https://github.com/wercker/getting-started-php/blob/master/index.php

Comment (hatchan) [inline content/quickstarts/building/php.md]: If this is going to be the new content from the sample app, then we need to make it valid JSON, by removing the double quotes and changing the single quotes to double quotes

File: content/quickstarts/building/php.md [hunk 2]
```diff
@@ -1,2 +1,339 @@
...
+#### Dev pipeline
+
+```yaml
+# The container definition we want to use for developing our app
+box: php
+# Defining the dev pipeline
+dev:
+  steps:
+    - install-packages:
+      packages: git
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: Add two more spaces to make it consistent with the other steps

File: content/quickstarts/building/php.md [hunk 3]
```diff
@@ -1,2 +1,339 @@
...
+### Building your app
+
+First, let's revisit our **wercker.yml** again.
+
+```yaml
+box: php
+dev:
+  steps:
+   - install-packages:
+      packages: git
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: + 2 indenting

File: content/quickstarts/building/php.md [hunk 4]
```diff
@@ -1,2 +1,339 @@
...
+
+### Building your app
+
+First, let's revisit our **wercker.yml** again.
+
+```yaml
+box: php
+dev:
+  steps:
+   - install-packages:
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: + 1 indenting

File: content/quickstarts/building/php.md [hunk 5]
```diff
@@ -1,2 +1,339 @@
...
+    - script:
+        name: install dependencies
+        code: composer install --no-interaction
+    - internal/watch:
+        code: php -S 0.0.0.0:8000
+        reload: true
+build:
+  steps:
+   - install-packages:
+      packages: git
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: \+2 indenting

File: content/quickstarts/building/php.md [hunk 6]
```diff
@@ -1,2 +1,339 @@
...
+        code: curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/local/bin --filename=composer
+    - script:
+        name: install dependencies
+        code: composer install --no-interaction
+    - internal/watch:
+        code: php -S 0.0.0.0:8000
+        reload: true
+build:
+  steps:
+   - install-packages:
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: \+1 indenting

File: content/quickstarts/building/php.md [hunk 7]
```diff
@@ -1,2 +1,339 @@
...
+listed here are **script** steps, which are bash scripts declared inline. We use
+these steps to install PHPUnit and Composer, after which we user Composer the dependencies
+as described in **composer.json**. Lastly, we serve our application so we can run our tests.
+
+#### wercker build
+
+Now that we have a better understanding of our **wercker.yml** let's go ahead
+and let wercker **build** our project. In your project folder, run:
+
+```
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: add `no-highlight`

File: content/quickstarts/building/php.md [hunk 8]
```diff
@@ -1,2 +1,339 @@
...
+
+Let's see these steps in action now and fire up our _dev pipeline_.
+
+### wercker dev
+
+In your project folder, run `wercker dev --publish 8000`. You should see
+something similar to the following output:
+
+```no-highlight
+--> Executing pipeline
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: add `$ wercker dev --publish 8000`

File: content/quickstarts/building/php.md [hunk 9]
```diff
@@ -1,2 +1,339 @@
...
+
+Now we have to choose who owns the app. For this tutorial, go ahead and select
+yourself. If you like, you can also select an organization you created on
+wercker. Click on **Use selected owner** once you're ready.
+
+#### Configure Access
+
+The next step is about configuring access, and the first option - **checkout
+the code without using an SSH key** - is fine for the purpose of this tutorial,
+because it's an open source and public application.
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: Maybe just say "because it's an public application"

File: content/quickstarts/building/php.md [hunk 10]
```diff
@@ -1,2 +1,339 @@
...
+Finally, once you've verified all the settings you can click **Finish** to
+complete setting up our app!  When done, you will be redirected to your very
+own app page, which looks empty now, so let's go ahead and change that.
+
+### Triggering your first build
+
+Wercker will automatically trigger a build every time you push new code to your
+Git provider. Let's see that in action. In your working directory, run
+
+```
```
Comment (hatchan) [inline content/quickstarts/building/php.md]: `no-highlight`

**Comment by hatchan** [comment]
Some minor comments

**Comment by Tonnu** [comment]
Thanks for the feedback @hatchan! @lindseybateman could you add a 'php.png' to this branch? Then we can merge it in.

**Comment by lindseybateman** [comment]
@Tonnu added the logo for php

File: docs-sidebar/toc-quickstarts.json [hunk 11]
```diff
@@ -4,7 +4,8 @@
     "golang.md",
     "javascript.md",
     "python.md",
-    "ruby.md"
+    "ruby.md",
+		"php.md"
```
Comment (hatchan) [inline docs-sidebar/toc-quickstarts.json]: There's some weird indenting

**Pull request merged**